### ISSUE TITLE
Module settings via the ui

### ISSUE THREAD

**Issue by iglocska** [MISP/MISP#2569]
Change proxy settings if modules via the ui

**Comment by frennkie** [comment]
It would be great if _all_ settings that people need would be accessible over the MISP admin menu.
Currently some things need to be configured in text/config files. One example are the settings for proxy access (which - ideally - should also be passed on to misp-modules somehow).

**Comment by seldem** [comment]
I agree on this and  I am in trouble for proxy to work inside modules. I could not find how to do it.
